pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -222,6 +222,7 @@ const (
 var failureInjectionMutators = []mutator{
 	panicNodeMutator{},
 	networkPartitionMutator{},
+	multiRegionSimulationMutator{},
```

Comment:
Wondering if this should be considered a failure injection mutator? It does utilize a failure injection to simulate the multi-region cluster but the mutator itself isn't exactly a failure injection..